src/cmds/mod.py

```diff
@@ -120,19 +120,19 @@
                     name="purge",
                     description="\"Purges\" or deletes messages from a channel in bulk.",
                     options=[
+                        Option(
+                            type=OptionType.INTEGER,
+                            name="amount",
+                            description="How many messages you wish to purge.",
+                            required=True,
+                        ),
                         Option(
                             type=OptionType.CHANNEL,
                             name="channel",
                             description="The channel you wish to slowmode.",
```

Comment:
```suggestion
                            description="The channel you wish to purge. Defaults to current channel",
```

also make channel everywhere not required pls